src/Features/Lsif/Generator/Generator.cs

```diff
@@ -160,13 +189,13 @@ public async Task GenerateForCompilationAsync(Compilation compilation, string pr
 
             var uri = syntaxTree.FilePath;
 
+            var text = await semanticModel.SyntaxTree.GetTextAsync(CancellationToken.None);
```

Comment:
Interesting, I feel like this method should be taking a cancellation token already, is it odd that it doesn't or is this expected?

Reply:
> I feel like this method should be taking a cancellation token already, is it odd that it doesn't or is this expected?

From a style perspective, yes it's odd there's no CancellationToken.

From a functionality perspective, it doesn't really matter, as this is a command line application. We cancel by killing the process.

Reply:
i have a separate PR that properly threads through cancellation.  It will allow us to support ctrl-c in a sensible fashion in the future.